### PR TITLE
Bug bash #2 (mostly in skeleton view)

### PR DESCRIPTION
**Pull request by lucyxubroad**
- Remove skeleton view from the search for now
- Added skeleton view to notifications and suggestions
- Cleaned up notifications and suggestions view
- Added timestamps for comments, notifications, and suggestions
- Fixed the skeleton view weird header loading thing

### CODE REVIEW

File: Flick/Cells/SuggestionTableViewCell.swift [hunk 1]
```diff
@@ -116,7 +129,7 @@ class SuggestionTableViewCell: UITableViewCell {
         }
 
         notificationLabel.snp.makeConstraints { make in
-           make.centerY.equalTo(profileImageView)
+            make.top.equalTo(profileImageView).offset(4)
```
Comment (haiyingweng) [inline Flick/Cells/SuggestionTableViewCell.swift:132]: lol tiny indent there

File: Flick/Controllers/LoginViewController.swift [hunk 2]
```diff
@@ -19,7 +19,7 @@ class LoginViewController: UIViewController {
     private let logoImageView = UIImageView()
 
     // MARK: - Private Data Vars
-    private let profileSize = CGSize(width: 50, height: 50)
+    private let profileSize = CGSize(width: 100, height: 100)
```
Comment (haiyingweng) [inline Flick/Controllers/LoginViewController.swift:22]: increasing the size might would make the image larger and base64 string even longer right

Comment (haiyingweng) [reply Flick/Controllers/LoginViewController.swift:22]: wonder if that's slow things down

**Review by haiyingweng — APPROVED** [review]
awesome

File: Flick/Controllers/LoginViewController.swift (hunk 2 above)
Comment (lucyxubroad) [reply Flick/Controllers/LoginViewController.swift:22]: Yea.. right now the quality is so bad tho

**Pull request merged**